ReactAndroid/src/main/java/com/facebook/react/views/text/ReactBaseTextShadowNode.java

```diff
@@ -304,6 +314,13 @@ private int getTextAlign() {
     return textAlign;
   }
 
+  @ReactProp(name = ViewProps.LETTER_SPACING, defaultFloat = Float.NaN)
+  public void setLetterSpacing(float letterSpacing) {
+    mPreviousLetterSpacing = letterSpacing;
```

Comment:
Nitpick (though I'm not an authority on this codebase): "previous" implies you're keeping an old state, but in fact you're just converting units for the _same_ state; maybe something like `mLetterSpacingPts` and `mLetterSpacingEms`? (or `mLetterSpacingInput` and `mLetterSpacing`, following the pattern seen elsewhere in this file with e.g. `mFontSize`)